### PR TITLE
Added js file

### PR DESCRIPTION
**Pull request by iomarcovalente**
Mind that I added a file for js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Happy Website Readme
+
+This is an happy Website
```
Comment (alicemirandola) [inline README.md]: you should add the name of the collaborators please

Comment (iomarcovalente) [reply README.md]: yes, sorry, you are right

Comment (iomarcovalente) [reply README.md]: will do immediately

**Review by alicemirandola — APPROVED** [review]
👍 💯  thanks

**Pull request merged**